sync/class.jetpack-sync-full.php

```diff
@@ -337,22 +336,35 @@ public function expand_updates( $args ) {
 		return $args;
 	}
 	
-	function update_sent_progress_action( $action_to_send ) {
+	function update_sent_progress_action( $actions_sent ) {
 		$modules_count = array();
 		$status = $this->get_status();
 		if ( is_null( $status['started'] ) || $status['finished'] ) {
 			return;
 		}
-		foreach( $action_to_send as $action ) {
+
+		if ( in_array( 'jetpack_full_sync_start', $actions_sent ) ) {
```

Comment:
This doesn't seem like it should work. Isn't `$actions_sent` an array of actions, i.e. something like:

```
array(
  'some_action_name',
  array( 
    /* some action args */
  ),
  $user_id,
  $timestamp
) 
```